rosie/src/main/java/com/karumi/rosie/view/RosieFragment.java

```diff
@@ -59,9 +59,7 @@
       Bundle savedInstanceState) {
     injectDependencies();
     int layoutId = getLayoutId();
-    View view = inflater.inflate(layoutId, container, false);
-    ButterKnife.bind(this, view);
-    return view;
+    return inflater.inflate(layoutId, container, false);
```

Comment:
I think the current implementation is good enough :) IMHO we can already merge this PR :)